src/transformers/file_utils.py

```diff
@@ -992,7 +998,10 @@ def cached_path(
     return output_path
 
 
-def http_get(url, temp_file, proxies=None, resume_size=0, user_agent: Union[Dict, str, None] = None):
+def http_user_agent(user_agent: Union[Dict, str, None] = None) -> str:
```

Comment:
(super nit) . Think `Optional[Union[Dict, str]]` would be cleaner than  `Union[Dict, str, None]`

Reply:
it's 4 characters shorter, and to be completely honest, I almost ran a regex search and replace to replace all occurences with Union[xxx, yyy, None] ;)

Reply:
I'll leave this stylistic point of decision to @LysandreJik and @sgugger though 😁